brave/brave-core
# [Rewards 3.0] Add connect account UI

Resolves https://github.com/brave/brave-browser/issues/39876
Resolves https://github.com/brave/brave-browser/issues/39852



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Note: QA may want to wait to test this until https://github.com/brave/brave-browser/issues/39877 is also complete.

* Enable the Rewards 3.0 flag: `#brave-rewards-new-rewards-ui` (Show the new Rewards UI)
* With a new profile, enable Rewards.
* On either the rewards page or rewards panel, press the "Connect account" button.
  * (If pressed on the panel, it should open up the Rewards page.)
* Verify that connect account screen is displayed with the correct options.
* Select a provider and complete the provider login flow.
* Ensure that user is in the connected state after login flow is completed.
